docs/administrator/security/component-permission.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Component Permissions
+---
+
+# Permissions for Karmada Components
+
+This document provides a detailed explanation of the Role-Based Access Control (RBAC) permissions required for key components of Karmada. It aims to ensure secure management and compliant access to resources within the system. As a multi-cluster management framework, controlling permissions between Karmada components is essential for maintaining the security and stability of the entire system. This guide will help administrators understand and configure the RBAC permissions needed for Karmada components effectively.
+
+For more detailed information about RBAC permissions, please refer to the [official RBAC documentation](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
+
+*Note: The resources in this document are all Karmada control plane resources.*
```

Comment:
Is that true?
For instance, the `leases` required by `karmada-operator` is Kubernetes instead of the Karmada control plane.

Reply:
Great catch! This note should be placed in the "karmada-agent" section.